src/main/scala/beam/agentsim/infrastructure/charging/ChargingPointType.scala

```diff
@@ -99,6 +99,12 @@ object ChargingPointType {
       case ChargingStationCcsComboType2 => 50
       case TeslaSuperCharger            => 135
       case CustomChargingPoint(_, v, _) => v
+      // legacy charging points (values taken from 2018 BEAM code)
```

Comment:
@johanneshiry plz double-check the values i put in place for the legacy types, if you don't mind - both AC/DC and the KwH. i just grabbed them from the old implementation. i think it should work as expected.

Reply:
The "worse" thing about the old definitions is, that Level1 / Level2 Charging rate are not fixed defined but more of a range of installed charging capacity. Anyway the values you picked are in range or close to the definitions of Level1/2 charging. So we can keep them for now until the personal ev branch will be merged in.